File: dayFour/java/AccountFunctions.java
public class AccountFunctions {

    public static double checkBalance(double balance) {
        return balance;
    }
    public static double deposit(double amount, double balance, String transactions) {
        double newBalance = 0;
        newBalance = balance + amount;
        
        transactions = "Deposited: ₦" + amount + "| New Balance: ₦" + balance;

        return newBalance;
    }
    public static double withdraw(double amount, double balance, String transactions) {
        double newBalance = 0;
        if(amount <= balance) {
            newBalance = balance - amount;
        }
        else{
            newBalance = balance;
        }
        return newBalance;
    }
    public static String showTransactions(String transactions){
        return transactions;
    }
}

File: dayFour/python/account_functions.py

def checkBalance( balance) :
    return balance;

def deposit(amount,balance, transactions) :
    newBalance = 0.0;
    newBalance = balance + amount;
    balance = newBalance;

    return newBalance;
    
def withdraw(amount,balance, transactions) :
    newBalance = 0;
    if amount <= balance :
        newBalance = balance - amount;
    else :
        newBalance = balance
    return newBalance;

def store_transaction(amount, balance):

    transactions = [amount, balance];


def showTransactions( transactions):

    return transactions;
        
    

